configuration/appsettings.rst

```diff
@@ -189,7 +189,7 @@ MongoDB
 ::
 
     "MongoDbOptions": {
-        "ConnectionString": "mongodb://localhost/vonkstu3",
+        "ConnectionString": "mongodb://localhost/vonkdata",
```

Comment:
When you are already changing the databasename, then this is the time to get rid of the name `vonk`, right?

Reply:
We would have to change the default appsettings as well then (it contains 'vonkdata' currently), both for MongoDB and for SQLite. Plus, for the SQLite admin db we also use vonkadmin.db, so we would have also have to change the pipeline + artifact. And if we have to keep MongoDB and SQLite 'vonk', then I prefer to align the SQL connection string examples as well (even though they are not in the appsettings.default).
So I agree that it would be cleaner, but it is too much of a change to do in this PR.

Reply:
I understand.